tests/listener/listener.go

```diff
@@ -0,0 +1,43 @@
+/*
+Copyright 2024 The Flux authors
```

Comment:
Please copy the header from an existing file, the formatting here is wrong.

Reply:
Fixed, thanks! This time I copied from a more recent file, `api/v1/helmchart_types.go`. Apparently only a few line breaks were missing. Can you please confirm?

Reply:
Looks Ok now, ty.